searx/webapp.py

```diff
@@ -659,8 +704,9 @@ def search():
     except SearxParameterException as e:
         logger.exception('search error: SearxParameterException')
         return index_error(output_format, e.message), 400
-    except Exception as e:
-        logger.exception('search error')
+    except Exception as e:  # pylint: disable=broad-except
+        logger.exception(e, exc_info=True)
+        logger.debug('search error')
```

Comment:
Is `logger.exception("search error")` not enough?

Reply:
For me? .. yes :-) .. I will remove the logger.debug message.

Reply:
fixed here:
- https://github.com/searxng/searxng/commit/e295bbf0679fe1def6983be969b735ef20d26c19#diff-ab9553a5df12bab91ccac7a4b9af78fbd160e35bffae9a340bf75ed736d50a07R320-R322
- https://github.com/searxng/searxng/commit/e295bbf0679fe1def6983be969b735ef20d26c19#diff-ab9553a5df12bab91ccac7a4b9af78fbd160e35bffae9a340bf75ed736d50a07L309
- https://github.com/searxng/searxng/commit/e295bbf0679fe1def6983be969b735ef20d26c19#diff-ab9553a5df12bab91ccac7a4b9af78fbd160e35bffae9a340bf75ed736d50a07R713-R714